### DISCUSSION TITLE
Two-factor authentication (2FA)[Feature] feature-name-goes-here

### DISCUSSION THREAD

**Discussion by Joberbaro** [immich-app/immich#14309] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Yesterday initiates the use of Immich's application, first of all I want to congratulate the Imich Desarrillator team for this wonderful tramp.

I wanted to check the authentic and security apparatus, I found that web application does not have the option to configure two-factor authentication (2FA), I do not know if the app has said configuration, perome it stops very important to have an additional security layer, due to the sensitivity of our information.
Look for something related to the zero knowledge encryption of information on Immich's public servers but it was not possible to find this information and it is another important appliance for data protection.

Greetings,

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by mmomjian** [comment]
Hello and welcome to immich! This feature has been requested in #8175 but the team has decided not to implement it. We offer Oauth integration for more advanced authentication.
